package.json

```diff
@@ -14,8 +14,8 @@
     "@cloudflare/workers-types": "^3.11.0",
     "@datadog/browser-logs": "^4.11.3",
     "@rocicorp/logger": "^2.2.0",
-    "@rocicorp/reflect": "^0.10.0",
-    "@rocicorp/reflect-server": "^0.18.0",
+    "@rocicorp/reflect": "file:rocicorp-reflect-0.11.0.tgz",
```

Comment:
I'm unclear on why we switch to a file?

Reply:
It's just to enable testing on a vercel preview / branch, we'd not check this in.